internal/charmstore/server.go

```diff
@@ -72,6 +73,10 @@ type ServerParams struct {
 	// that an HTTP request will wait for a free connection
 	// when the MaxConcurrentHTTPRequests limit is reached.
 	HTTPRequestWaitDuration time.Duration
+
+	// AuditLogger optionally holds the logger which will be used to
+	// write audit log entries.
+	AuditLogger *lumberjack.Logger
```

Comment:
Because we want to have rolling files instead of one growing giant file and lumber jack seems to be the natural choice.